### DISCUSSION TITLE
Settings

### DISCUSSION THREAD

**Discussion by bertelu** [amanharwara/altus#148] [Q&A]
don't know why but seems that the top menu disappeared. 
how to re-enable it and so enter into settings?

thanks

Luca

**Comment by amanharwara** [comment]
You can press `Ctrl`+`,` to open the settings.

**Comment by HikeAndMap** [comment]
Could I suggest - since  ctrl + , isn't the easiest to remember add this option to enable/disable in the tray popup-menu?

**Reply by amanharwara** [reply]
Good suggestion. I'll implement this.

**Reply by amanharwara** [reply]
Implemented in: https://github.com/amanharwara/altus/releases/tag/4.4.4
